Resources/Textures/Objects/Fun/Instruments/banjo.rsi/meta.json

```diff
@@ -17,6 +17,16 @@
       {
         "name": "inhand-right",
         "directions": 4
+      },
+      {
+        
+        "name": "equipped-SUITSTORAGE",
+        "directions": 4,
+        "comment": "lol i made these shitty sprites in like 2 minutes -archangel"
```

Comment:
This `comment` field is not defined within the [RSI spec](https://docs.spacestation14.com/en/specifications/robust-station-image.html); additional credits should be added to the copyright field at the root of the json object